syncano/models/manager.py

```diff
@@ -828,6 +828,26 @@ def _get_endpoint_properties(self):
         return self.model._meta.resolve_endpoint(self.endpoint, defaults), defaults
 
 
+class RuntimeChoices(object):
```

Comment:
Regarding your example. You assume that developer will know that there is such a thing like RUNTIMES.PHP, which is not a case because there is no such a thing in a code. First, he has to inspect it somehow, to get to know what are the runtimes. In case of python and nodejs code this will not work either. Some of the codeboxes/scripts still have, a 'nodejs' and 'python' in runtime_name, and they are not listed in runtime list endpoint, so validation will raise an exception. Just providing constants in source is IMHO better and shorter solution.